pkg/cmd/openshift-sdn-node/kube_proxy.go

```diff
@@ -142,6 +142,7 @@ func newProxyServer(config *kubeproxyconfig.KubeProxyConfiguration, client clien
 			recorder,
 			healthzServer,
 			config.NodePortAddresses,
+			false, //@DanWinship can you take a look here to make sure im using the right value for initOnly
```

Comment:
yes.
(If we weren't about to kill openshift-sdn off, then we could take advantage of the new `--init-only` feature, but it's not worth it at this point.)